src/main/java/com/pileproject/drivecommand/command/CommandBase.java

```diff
@@ -32,27 +35,28 @@ public CommandBase(CommandType type, Map<String, Object> args) {
     }
 
     /**
-     * Get its command type
+     * Get this command type.
```

Comment:
General comment:
Javadoc comment for methods are usually start with third-person singular verb
http://www-inf.int-evry.fr/cours/java/JAVADOC/writingdoccomments.html
example: https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html